product_info.toml

```diff
@@ -1,36 +1,36 @@
 title = "All known products that require license acceptance, and the mapping of parent to child relationships"
 
 [[products]]
-name = "chef-infra"
-pretty_name = "Chef Infra"
+id = "infra-client"
+pretty_name = "Chef Infra Client"
 hab_pkg_id = "chef/chef-client"
-filename = "chef_infra"
+filename = "infra_client"
```

Comment:
Similarly I figured it was okay to change the filename since we have not released the products yet (besides InSpec, whose filename is not changing).

Any issues with this? If we decide to change the filename again in the future I will add logic to handle changing existing license marker files with some migration logic (EG, the library would see `infra_client` in the accepted licenses and automatically rename that file to whatever new filename we picked).